GoogleForCreators/web-stories-wp
# Dashboard: Applied design updates to Travel template

## Summary

This PR applies the new design updates to the Travel template (and removes old test animations applied to DIY template which were throwing prop errors).

## User-facing changes

Layout and text sizes have been update on the Travel template based off the recommendations found in this [figma design doc](https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61).

## Testing Instructions

1.) Go to "Explore Templates".
2.) Hover over the Travel template and click on "View".
3.) Take a look at each template page in the detail view for Travel.  Make sure images are showing up and the layout appears as shown in the figma updates.

---

Fixes #2475 

## Screenshots from Editor

<img src="https://user-images.githubusercontent.com/40646372/84953818-12d2fc00-b0a9-11ea-9af1-4c3ec04ac8c9.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/84953845-1ebebe00-b0a9-11ea-9583-94400f268406.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/84953857-27af8f80-b0a9-11ea-9bbd-18a931fb2800.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/84953957-575e9780-b0a9-11ea-9b83-1c25e25fe047.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/84953972-5fb6d280-b0a9-11ea-9e15-b4ad15d43726.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/84953986-680f0d80-b0a9-11ea-8066-1d3aa0ba33da.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/84954006-71987580-b0a9-11ea-8e09-62fc35d90cb8.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/84954015-79581a00-b0a9-11ea-9544-d77cfede8309.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/84954033-8117be80-b0a9-11ea-83cc-82b2a198249f.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/84954054-8a089000-b0a9-11ea-8891-c8c3629dcca4.png" height="200">
